ci-operator/config/openshift/cincinnati/openshift-cincinnati-master.yaml

```diff
@@ -8,27 +8,30 @@ base_images:
     namespace: openshift
     tag: golang-1.13
 build_root:
-  image_stream_tag:
-    name: cincinnati-build-root
-    namespace: cincinnati-ci
-    tag: buildroot
+  project_image:
+    dockerfile_path: dist/Dockerfile.build/Dockerfile
+    inputs:
+      centos:
+        as:
+        - quay.io/app-sre/centos:7
```

Comment:
Hrm, this is not getting replaced in [the `Dockerfile`][1]; [images][2]:

```
2020/12/23 17:24:39 Build root failed, printing logs:
Cloning "https://github.com/openshift/cincinnati.git" ...
	Commit:	2928002d5110d8bbeedd252cbe19c50c0327bc48 (Merge pull request #350 from LalatenduMohanty/remove_unreadable_file)
	Author:	OpenShift Merge Robot <openshift-merge-robot@users.noreply.github.com>
	Date:	Tue Dec 22 16:05:55 2020 -0500
Caching blobs under "/var/cache/blobs".
Pulling image quay.io/app-sre/centos:7 ...
```

I'm still not clear on the distinction between `from` in a ci-operator config and an `inputs` entry whose `as` matches the `FROM`.  But double checking [my claims about `from`][3] with the current ci-tools tip:

```console
$ git blame pkg/api/types.go | grep project_image
9443d98fc5 (Steve Kuznetsov          2018-09-20 16:06:08 -0700  330)    ProjectImageBuild       *ProjectDirectoryImageBuildInputs `json:"project_image,omitempty"`
0f4249e1f4 (Nikolaos Moraitis        2018-08-31 18:18:07 +0200 1193) type ProjectDirectoryImageBuildInputs struct {
d109fcd44e (Steve Kuznetsov          2018-01-31 14:30:58 -0800 1194)    // ContextDir is the directory in the project
d109fcd44e (Steve Kuznetsov          2018-01-31 14:30:58 -0800 1195)    // from which this build should be run.
a634cbbab2 (Steve Kuznetsov          2019-02-26 14:47:49 -0800 1196)    ContextDir string `json:"context_dir,omitempty"`
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1197) 
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1198)    // DockerfilePath is the path to a Dockerfile in the
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1199)    // project to run relative to the context_dir.
a634cbbab2 (Steve Kuznetsov          2019-02-26 14:47:49 -0800 1200)    DockerfilePath string `json:"dockerfile_path,omitempty"`
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1201) 
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1202)    // Inputs is a map of tag reference name to image input changes
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1203)    // that will populate the build context for the Dockerfile or
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1204)    // alter the input image for a multi-stage build.
a634cbbab2 (Steve Kuznetsov          2019-02-26 14:47:49 -0800 1205)    Inputs map[string]ImageBuildInputs `json:"inputs,omitempty"`
bd7f9b1203 (Clayton Coleman          2018-06-06 18:25:35 -0400 1206) }
```

So yeah, still no `from` entry.  I dunno why the initial `FROM` entry needs special handling, but will root around in ci-tools to see if I can understand the motivation.

[1]: https://github.com/openshift/cincinnati/blob/2928002d5110d8bbeedd252cbe19c50c0327bc48/dist/Dockerfile.build/Dockerfile#L1
[2]: https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_release/14518/rehearse-14518-pull-ci-openshift-cincinnati-master-images/1341796606067871744#1:build-log.txt%3A17
[3]: https://github.com/openshift/release/pull/14518#issue-544929615

Reply:
Pivoted to a CentOS root with https://github.com/openshift/release/pull/14518#discussion_r548152708